docs/en/stack/ml/get-started/ml-getting-started.asciidoc

```diff
@@ -1,5 +1,8 @@
 [[ml-getting-started]]
 == Getting started with {ml}
+:keywords: {ml-init}, {es}, {anomaly-detect}, tutorial
```

Comment:
Though I initially put "Elasticsearch" as the product, I'm incline to change it to "Elastic Stack" since most of these pages assume you're using Kibana to view the results
```suggestion
:keywords: {ml-init}, {stack}, {anomaly-detect}, tutorial
```